build-aux/m4/bitcoin_qt.m4

```diff
@@ -257,9 +260,9 @@ dnl ----
 
 dnl Internal. Check included version of Qt against minimum specified in doc/dependencies.md
 dnl Requires: INCLUDES must be populated as necessary.
-dnl Output: bitcoin_cv_qt5=yes|no
-AC_DEFUN([_BITCOIN_QT_CHECK_QT5],[
-  AC_CACHE_CHECK(for Qt 5, bitcoin_cv_qt5,[
+dnl Output: bitcoin_cv_qt_minimumrequired=yes|no
+AC_DEFUN([_BITCOIN_QT_CHECK_MINIMUM_REQUIRED],[
+  AC_CACHE_CHECK(for Qt >= 5.5.1, bitcoin_cv_qt_minimumrequired,[
```

Comment:
non-blocking nit: here and in other places mind quoting the first argument:
```suggestion
  AC_CACHE_CHECK([for Qt >= 5.5.1], bitcoin_cv_qt_minimumrequired,[
```
?